tsdb/engine/tsm1/engine_test.go

```diff
@@ -17,7 +17,7 @@ import (
 // Ensure an engine containing cached values responds correctly to queries.
 func TestDevEngine_QueryCache_Ascending(t *testing.T) {
 	// Generate temporary file.
-	f, _ := ioutil.TempFile("", "tsm1dev")
+	f, _ := ioutil.TempFile("", "tsm")
```

Comment:
Some test code uses "tsm" as the extension, other code uses "tsm1". Let's standardize, right?